apps/web/src/components/ContractButton.tsx

```diff
@@ -12,18 +14,21 @@ export const ContractButton = ({
   ...rest
 }: ContractButtonProps) => {
   const { address } = useAccount()
-  const { chain } = useNetwork()
+  const { chain: wagmiChain } = useNetwork()
+  const chain = useChainStore((x) => x.chain)
```

Comment:
a nit: can we use more explicit variables?

Reply:
updated!